codespell_lib/data/dictionary.txt

```diff
@@ -1680,6 +1685,7 @@ alternaive->alternative
 alternaives->alternatives
 alternarive->alternative
 alternarives->alternatives
+alternatiely->alternately
```

Comment:
I believe `alternatively` has a higher probability than `alternately`.
```text
alternatiely->alternatively, alternately,
alternatie->alternatives, alternate,
alternaties->alternatives, alternates,
```
https://github.com/search?q=alternatie&type=code
https://github.com/search?q=alternaties&type=code

Reply:
[bc58273](https://github.com/codespell-project/codespell/pull/2370/commits/bc5827378dc7f488c617b9163ed1940d76a65bbd)

Reply:
@luzpaz Please help to resolve this again. You missed `alternatiely` modification. Thank you very much.
```text
alternatiely->alternatively, alternately,
```

Reply:
There was a typo in your original comment: 
> alternatie->alternative**s**, alternate,

So that through me off. Sorry for missing that entry. Fixed in [d95b166](https://github.com/codespell-project/codespell/pull/2370/commits/d95b166d8bebed8cf0188e0cf955534bec226716)